data/human/human missions.txt

```diff
@@ -6849,3 +6849,318 @@ mission "FW Refugees to Humanika"
+
+event "adelita 2 offer expired"
+
+mission "Adelita 2"
+	name "Film Syndicate sweatshop on <planet>"
+	description "Participate in an unauthorized filming of a Syndicate sweatshop on <destination>. (You will be paid <payment> after you complete the entire job.)"
+	cargo "stealth drones" 1
+	passengers 3
+	"apparent payment" 200000
+	source "Moonshake"
+	destination "Maker"
+	to offer
+		has "Adelita 1: done"
+		not "adelita 2 offer expired"
+
+	on offer
+		conversation
+			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat. Before your meal arrives, a woman with dark hair, maybe in her late thirties, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of or can even pronounce.`
+			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So Captain <last>, what did you think of the exhibition?"`
+			choice
+				`	"I found it quite moving."`
+				`	"It was OK."`
+					goto wasok
+				`	"It was too political for me."`
+					goto political
+			`	"I'm glad you liked it," she says, smiling.`
+				goto choice
+
+			label wasok
+			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says, smiling.`
+				goto choice
+
+			label political
+			`	"Oh. Unfortunately for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it anyway without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installations," she says with a smile. She gets her smoothie to-go and walks out.`
+				decline
+
+			label choice
+			`	"Mariel said you saw me take the photos of the woman and her baby," she says. "I really like that moment, but not many notice it." She leans in closer. "We need a captain to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. I can't tell you much until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows. Oh, and we have to leave today. It will pay, of course. <payment>. So what do you say? Are you in or out?" As she finishes, your order is served while she drinks her smoothie.`
```

Comment:
```suggestion
			`	"I head from Mariel that you saw me take the photos of the woman and her baby," she says. "I really liked that moment, but not many notice it." She gives a quick glance around before leaning in closer. "We need a captain to transport a team to one planet, quickly land on another, then pick up the team and return here with everyone safe and sound. I can't tell you much until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows?" She looks at you expectantly.`
			`	"Oh, and we have to leave today. It will pay, of course. <payment>. So what do you say? Are you in or out?" As she finishes, your order is served while she drinks her smoothie.`
```